code/modules/shuttle/emergency.dm

```diff
@@ -226,24 +235,31 @@
 				mode = SHUTTLE_IDLE
 				timer = 0
 		if(SHUTTLE_CALL)
+			if(time_left <= 160 && crash && !crash_announced)
+				//Notify people that a coder is an absolute ass
+				event_announcement.Announce("An error has been detected in shuttle docking systems - brace for impact!", "Don't drink and drive", new_sound = 'sound/misc/notice1.ogg')
+				crash_announced = TRUE
 			if(time_left <= 0)
 				//move emergency shuttle to station
 				if(dock(shuttle_master.getDock("emergency_home")))
 					setTimer(20)
 					return
+				//Be a dick to everyone and ruin their day (okay, not everyone, only those who are not buckled)
+				if(crash)
+					var/area/target = locate(crash_target)
+					if(target)
+						for(var/mob/M in range(16, target))
```

Comment:
Why would you do it this way (potentially affecting things in space and ghosts/observers) rather than `for(var/mob/living/M in get_areaname("escape"))` to only affect living-typed mobs in the specified area?

Reply:
Oooh, I messed this up. Hold.